src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const initialKeys = Object.keys(robot);
+
+  for (let i = 0; i < initialKeys.length; i++) {
+    for (let j = i + 1; j < initialKeys.length; j++) {
+      if (robot[initialKeys[i]] === robot[initialKeys[j]]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    const reversedKey = robot[key];
+
+    delete robot[key];
```

Comment:
You are modifying the input object which is generally a bad practice. It is better to create a new object for the output.